Showroom/DribbbleShots/DribbbleShotsCollectionViewLayout.swift

```diff
@@ -8,94 +8,34 @@
 
 import UIKit
 
+private let kProposedSectionInset: CGFloat = 21
```

Comment:
It's obj-c style for constants, better put this constant inside DribbbleShotsCollectionViewLayout
example : 

`final class DribbbleShotsCollectionViewLayout: UICollectionViewFlowLayout {
    
    private enum Const {
        static let sectionInset: CGFloat = 21
    }
 `